rsconnect/main.py

```diff
@@ -1013,6 +1013,11 @@ def _warn_on_ignored_requirements(directory: str, requirements_file_name: str):
     is_flag=True,
     help='Force generating "requirements.txt", even if it already exists.',
 )
+@click.option(
+    "--package-installer",
```

Comment:
I think we discussed this elsewhere, but having "package installer" here and then "package manager" in the source code is a little confusing.